exporters/pelorus/__init__.py

```diff
@@ -47,7 +47,9 @@ def _print_version():
 # region: logging setup
 def _setup_logging():
     _print_version()
-    loglevel = os.getenv("LOG_LEVEL", DEFAULT_LOG_LEVEL).upper()
+    loglevel = utils.get_env_var("LOG_LEVEL", DEFAULT_LOG_LEVEL)
+    if isinstance(loglevel, str):
+        loglevel = loglevel.upper()
```

Comment:
In case where LOG_LEVEL is missing from the env, then the return value is None, which isn't a string, so it fails:

`AttributeError: 'NoneType' object has no attribute 'upper'
`

Reply:
But we give it a default value right above, don't we?